build-support/bin/release.sh

```diff
@@ -116,7 +116,7 @@ function pkg_testutil_install_test() {
   shift
   local PIP_ARGS=("$@")
   pip install "${PIP_ARGS[@]}" "pantsbuild.pants.testutil==${version}" && \
-  python -c "import pants.testutil"
+  python -c "import pants.testutil.option_util"
```

Comment:
This doesn't actually trigger an error on the `master` branch, for some reason. It imports just fine. I'm not sure why or how to reproduce in `release.sh`, even though I reproduce everywhere else like running `pex testutil==2.0.0a1` then `import pants.testutil.option_util`.